packages/deployments/contracts/contracts_forge/Connext.t.sol

```diff
@@ -53,4 +55,34 @@ contract ConnextTest is ForgeHelper {
       _recipient
     );
   }
+
+  // ============ setMaxRouters ============
+
+  // Should work
+  function testSetMaxRouters() public {
+    require(connext.maxRouters() != 10);
+
+    connext.setMaxRouters(10);
+    assertEq(connext.maxRouters(), 10);
```

Comment:
need to fix this, should be maxRoutersPerTransfer now

sorry

Reply:
Fixed here https://github.com/connext/nxtp/pull/916/commits/f57ccea96c97a0831cfe498ab6ac7cabc7d59787